spk/salt-minion/Makefile

```
@@ -16,26 +21,31 @@ UNSUPPORTED_ARCHS = $(PPC_ARCHS) $(ARMv5_ARCHS)
 # pip freeze > requirements.txt
 WHEELS = src/requirements.txt
 
-MAINTAINER = SynoCommunity
-DESCRIPTION = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
-RELOAD_UI = yes
+MAINTAINER   = SynoCommunity
+DESCRIPTION  = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
 DISPLAY_NAME = Salt Minion
-CHANGELOG = "Update to 3001.1"
+CHANGELOG    = "Update to 3003.3"
 
-HOMEPAGE   = https://www.saltstack.com/
-LICENSE    = Apache
+HOMEPAGE     = https://www.saltstack.com/
+LICENSE      = Apache
+LICENSE_FILE = src/LICENSE
 
 POST_STRIP_TARGET = salt-minion_extra_install
 
 # Service configuration
 CONF_DIR = src/conf
-SERVICE_USER = auto
 SERVICE_SETUP = src/service-setup.sh
 STARTABLE = yes
 
+SPK_USR_LOCAL_LINKS = bin:env/bin/salt-call
+SPK_USR_LOCAL_LINKS = bin:env/bin/salt-minion
+SPK_USR_LOCAL_LINKS = bin:env/bin/salt-proxy
+SPK_USR_LOCAL_LINKS = bin:env/bin/spm
+
```

Only the last (`spm`) command is linked
Using `+=` fixes this

```suggestion
SPK_USR_LOCAL_LINKS = bin:env/bin/salt-call
SPK_USR_LOCAL_LINKS += bin:env/bin/salt-minion
SPK_USR_LOCAL_LINKS += bin:env/bin/salt-proxy
SPK_USR_LOCAL_LINKS += bin:env/bin/spm
```

or alternatively you can use 

```Make
SPK_COMMANDS = env/bin/salt-call
SPK_COMMANDS += env/bin/salt-minion
SPK_COMMANDS += env/bin/salt-proxy
SPK_COMMANDS += env/bin/spm
```
```

---

Fixed.